core/src/specialized_solver/fast_monodomain_solver/fast_monodomain_solver_compute.tpp

```diff
@@ -178,12 +178,25 @@ void FastMonodomainSolverBase<
   // neighbors
   fiberPointBuffersStatesAreCloseToEquilibrium_[0] = active;
   fiberPointBuffersStatesAreCloseToEquilibrium_[nPointBuffers - 1] = active;
+  
+  int fiberDataNo = 0;
+  int pointBuffersNoLastFiberDataNo = 0;
```

Comment:
I do not immediately understand what this means.

Reply:
I renamed `pointBuffersNoLastFiberDataNo` to  `pointBuffersNoAtFiberStart`. 
@MakisH is this easier to understand?